README.md

```diff
@@ -54,6 +56,18 @@ RNA-seq:
      -profile rnaseq,<docker/singularity/podman/shifter/charliecloud/conda/institute>
 ```
 
+with limma+voom
+
+```bash
+ nextflow run nf-core/differentialabundance \
+     --input samplesheet.csv \
+     --contrasts contrasts.csv \
+     --matrix assay_matrix.tsv \
+     --gtf mouse.gtf \
+     --outdir <OUTDIR>  \
+     -profile rnaseq_limma,<docker/singularity/podman/shifter/charliecloud/conda/institute>
+```
+
```

Comment:
Could you update the note below? [Limma Voom cannot handle the offset matrix](https://bioconductor.org/packages/devel/bioc/vignettes/tximport/inst/doc/tximport.html#limma-voom) so users **must** use the second option of supplying the scaled matrices.

Reply:
@pinin4fjords Thank you for noticing that. I've added appropriate comments.